LV_Z_DISPLAY_DEV_NAME symbol has not got "parent" symbol with a type

## Description

Using stm32f429i_disc1 board with Zephyr 3.1.0 I got an error on CMake configuration step. This error is connected to absent of LVGM symbols in project by default. I mean, [LV_Z_DISPLAY_DEV_NAME ](https://github.com/zephyrproject-rtos/lvgl/blob/aae1c11102138cf596687e662c0d54b72f72bf9b/zephyr/Kconfig.graphical#L12) is supposed to be declared in LVGL module with an appropriate type, but it's not, because I haven't got LVGL module in my project.

### To Reproduce

Steps to reproduce the behavior:
1. Get Zephyr 3.1.0
2. `west build -b stm32f429i_disc1 samples/hello_world`

### Expected behavior
Project is configured and built.

### Actual behavior
```
warning: LV_Z_DISPLAY_DEV_NAME (defined at boards/arm/stm32f429i_disc1/Kconfig.defconfig:17) defined without a type
Parsing /workspace/project-example/zephyr/Kconfig
Loaded configuration '/workspace/project-example/zephyr/boards/arm/stm32f429i_disc1/stm32f429i_disc1_defconfig'
Merged configuration '/workspace/project-example/zephyr/samples/hello_world/prj.conf'

error: Aborting due to Kconfig warnings

CMake Error at /workspace/project-example/zephyr/cmake/modules/kconfig.cmake:290 (message):
  command failed with return code: 1
Call Stack (most recent call first):
  /workspace/project-example/zephyr/cmake/modules/zephyr_default.cmake:121 (include)
  /workspace/project-example/zephyr/share/zephyr-package/cmake/ZephyrConfig.cmake:51 (include)
  /workspace/project-example/zephyr/share/zephyr-package/cmake/ZephyrConfig.cmake:76 (include_boilerplate)
  CMakeLists.txt:5 (find_package)


-- Configuring incomplete, errors occurred!
FATAL ERROR: command exited with status 1: /usr/bin/cmake -DWEST_PYTHON=/usr/bin/python3 -B/workspace/project-example/build -S/workspace/project-example/zephyr/samples/hello_world -GNinja -DBOARD=stm32f429i_disc1 -DCMAKE_EXPORT_COMPILE_COMMANDS=ON
```

**Environment:**
 - OS: Linux
 - Toolchain: [Zephyr SDK 0.15.0-beta2](https://github.com/zephyrproject-rtos/sdk-ng/releases/tag/v0.15.0-beta2)

--- a/modules/Kconfig.lvgl
+++ b/modules/Kconfig.lvgl
@@ -9,6 +9,15 @@ config LVGL
 
 if LVGL
 
+config LV_Z_DISPLAY_DEV_NAME
+	string
+
+config LV_Z_HOR_RES_MAX
+	int
+
+config LV_Z_VER_RES_MAX
+	int
+
 config LV_Z_POINTER_KSCAN
 	bool
 

